beta/src/content/apis/react-dom/unmountComponentAtNode.md

```diff
@@ -2,11 +2,11 @@
 title: unmountComponentAtNode
 ---
```

Comment:
don’t forget to remote WIP flag from sidebar file

Reply:
Nice catch, done